pkg/ioutils/copy.go

```diff
@@ -0,0 +1,21 @@
+package ioutils
```

Comment:
But it's not pool, there is no Put and Get :) It just uses an underlying pool of specific size. Do you want me to move that pool there?